test/README.md

```diff
@@ -1,7 +1,29 @@
+This folder contains testing infrastructure for the pipelines in the skylab repository.
```

Comment:
Could we add a little more description here for someone who should wonder in? What does testing infrastructure imply? Some questions we could answer in this overview that I might need your help:
1) What are the tests for/what do they actually do?
2) How do these tests differ from those that are used for validation? For example, how are the configuration files used here different than those we present on Terra?

Reply:
Updated